core/src/test/scala/org/apache/spark/SecurityManagerSuite.scala

```diff
@@ -395,9 +397,54 @@ class SecurityManagerSuite extends SparkFunSuite with ResetSystemProperties {
     assert(keyFromEnv === new SecurityManager(conf2).getSecretKey())
   }
 
+  test("Use executor-specific secret file configuration.") {
```

Comment:
All test names in this class start with lower case, so please follow the pattern.